docs/app/assets/css/main.css

```diff
@@ -32,3 +32,28 @@ html[data-module="ui-pro"] .ui-only,
 html[data-module="ui"] .ui-pro-only {
   display: none;
 }
+
+/* For better modals stacking visual experience */
```

Comment:
What happens by default if we don't specify these? Just an idea but maybe we could add a `stacked` prop to the UModal component as well to use those animations (which would be defined inside `keyframe.css`)?